pkg/pillar/scripts/device-steps.sh

```diff
@@ -452,8 +465,9 @@ if [ ! -s "$DEVICE_CERT_NAME" ]; then
     sleep 10
     sync
     blockdev --flushbufs "$CONFIGDEV"
-    echo "$(date -Ins -u) Making /config read-only again"
-    mount -o remount,flush,ro /config
+    echo "$(date -Ins -u) Making persist /config read-only again"
+    mount -o remount,ro $CONFIGDIR
```

Comment:
Is the flush option a no-op?

Reply:
CONFIGDIR points to tmpfs at this point, flush is not implemented and mount complains